src/common/certificate-authorities/system-ca.test.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import https from "https";
+import { DSTRootCAX3 } from "./is-cert-active";
+import assert from "assert";
+import { describeIf } from "../../test-utils/skippers";
+import { platform } from "process";
+import { getDiForUnitTesting } from "../../main/getDiForUnitTesting";
+import injectSystemCAsOnMacOSInjectable from "./inject-system-cas-on-macos.injectable";
+import injectSystemCAsOnWindowsInjectable from "./inject-system-cas-on-windows.injectable";
+
+describe("CA tests", () => {
+  let injectSystemCAsOnMacOS: () => Promise<void>;
+  let injectSystemCAsOnWindows: () => Promise<void>;
```

Comment:
1. Would appreciate behavioural unit tests for modernisations such as in this PR. After all, this is the cheapest place to produce them, as now you, the dev, know a lot of all the moving parts being migrated to new injectables.

2. Maybe the said modernisation is even a more meaningful title for this PR :)

3. Remember: just integration testing this shouldn't be enough, as it serves other purpose.